### PR TITLE
CSD-304 Use views field view to make distinct resource results

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Use views_field_view to produce distinct results
- Adjust the view and include styling for resource list.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. make sure the stack repo has been updated, pull the latest and `composer install -n`
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-304_distinct`
1. `blt drupal:sync --site=cardinalservice`
1. view the `/resources` page and verify there are no duplicates.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
         "drupal/search_api": "^1.17",
         "drupal/viewfield": "^3.0@beta",
         "drupal/views_bulk_edit": "^2.4",
+        "drupal/views_field_view": "^1.0@beta",
```
Comment (joegl) [inline composer.json:87]: Are you comfortable using this module in beta? I know it's pretty widely used and I've used it on a number of D7 sites before, but not in 8.

File: config/sync/views_field_view.settings.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+evil: false
```
Comment (joegl) [inline config/sync/views_field_view.settings.yml:1]: ಠ_ಠ

**Comment by joegl** [comment]
It does look like there is an .scss merge conflict too.

File: composer.json (hunk 1 above)
Comment (pookmish) [reply composer.json:87]: yeah. its been pretty actively maintained

**Pull request merged**